themes/bootstrap3/js/facets.js

```diff
@@ -1,5 +1,81 @@
 /*global VuFind */
-/*exported initFacetTree */
+/*exported initFacetTree, setupFacetList */
+
+function overrideHref(selector, overrideParams = {}) {
+  $(selector).each(function overrideHrefEach() {
+    let dummyDomain = 'https://www.example.org'; // we need this since the URL class cannot parse relative URLs
```

Comment:
Is it possible you can use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/toString) here to avoid this hack? (I was not aware of it until @EreMaijala recently mentioned it on another PR...).

Reply:
I think that neither URL nor URLSearchParams is 100% suitable what needs to be achieved here.

The kind of URL that needs to be modified here: 
`/Search/Results?type=AllFields&amp;filter%5B%5D=language%3A%22German%22`

URL only supports a full URL like this:
`http://www.example.com/Search/Results?type=AllFields&amp;filter%5B%5D=language%3A%22German%22`

URLSearchParams only supports the query params after the ?:
`type=AllFields&amp;filter%5B%5D=language%3A%22German%22`

So either it is necessary to add the domain & remove it afterwards (as done in the current state with URL) or strip the Controller Action, then modify it using UrlSearchParams and add the Controller Action again afterwards. If you think that's more safe please let me know, then i will change it, but it seems like in both cases there is no way to modify it safely without any additional string processing.